src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEMethodSymbol.cs

```diff
@@ -788,6 +788,24 @@ private bool IsValidExtensionMethodSignature()
                 this.ParameterRefKinds.IsDefault && // No 'ref' or 'out'
                 !this.IsParams();
 
+        private bool IsValidUserDefinedOperatorIs()
+        {
+            foreach (var parameter in this.Parameters)
+            {
+                if (parameter.RefKind != ((parameter.Ordinal == 0) ? RefKind.None : RefKind.Out))
+                {
+                    return false;
+                }
+            }
+
+            return
+                (this.ReturnsVoid || this.ReturnType.SpecialType != SpecialType.System_Boolean) &&
+                !this.IsGenericMethod &&
+                !this.IsVararg &&
+                this.ParameterCount > 0 &&
```

Comment:
The only point of disagreement we had was a successful consumption of ```is``` operators declared in metadata that have Nullable<T> as their first parameter, even though declaring such an operator in source is an error. However, after off-line discussion, it became apparent that we could still end up with a Nullable<T> as the first parameter via a generic substitution on a valid ```is``` declared in source. And we will and want to consume such an operator successfully. Given that, the inconsistency around consuming  declarations from metadata doesn’t feel that unpleasant.